src/router.js

```diff
@@ -305,18 +307,19 @@ router.beforeEach((to, from, next) => {
     if (params.get("data-with-logo")) {
       store.setIframeHeaderCollapse(params.get("data-with-logo"))
     }
+
+    for (const themeType of Object.values(ThemeType)) {
+      if (params.get(themeType)) {
+        app.config.globalProperties.$theme.update(themeType)
```

Comment:
:scream:

C'est pas équivalent à 
```suggestion
        app.$theme.update(themeType)
```

Reply:
> 

Ce n'est pas équivalent car ici l'objectif est de vérifier que le thème fourni par l'utilisateur de l'iframe existe bien chez nous et d'update le thème uniquement si cette condition est vérifiée